src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -45,5 +55,23 @@ public class LogsConfiguration : ConfigurationCategory
         [DefaultValue(false)]
         [ConfigurationEntry("Logs.Compress", ConfigurationEntryScope.ServerWideOnly)]
         public bool Compress { get; set; }
+
+        #region Microsoft Logs
+        [Description("Will determine whether to disabale the microsoft logs")]
+        [DefaultValue(true)]
+        [ConfigurationEntry("Logs.Microsoft.Enable", ConfigurationEntryScope.ServerWideOnly)]
+        public bool DisableMicrosoftLogs { get; set; }
+        
+        [Description("The path to the folder where microsoft log will be written")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Logs.Microsoft.Path", ConfigurationEntryScope.ServerWideOnly)]
+        public PathSetting MicrosoftLogPath { get; set; }
+        
+        [Description("The path to json configuration file of microsoft logs")]
+        [DefaultValue("microsoftLogSettings.json")]
```

Comment:
Do not like the file name.
`settings.logs.microsoft.json` ?

Reply:
Done